src/modules/handlehost.cpp

```diff
@@ -47,7 +47,7 @@ struct HandleImpl
         this->initHandle->mem.mcpu.maxSrcSize = (RppiSize *)malloc(sizeof(RppiSize) * this->nBatchSize);
         this->initHandle->mem.mcpu.maxDstSize = (RppiSize *)malloc(sizeof(RppiSize) * this->nBatchSize);
         this->initHandle->mem.mcpu.roiPoints = (RppiROI *)malloc(sizeof(RppiROI) * this->nBatchSize);
-        this->initHandle->mem.mcpu.scratchBufferHost = (Rpp32f *)malloc(sizeof(Rpp32f) * 99532800 * this->nBatchSize); // 7680 * 4320 * 3
+        this->initHandle->mem.mcpu.scratchBufferHost = (Rpp32f *)malloc(sizeof(Rpp32f) * 99532800 ); // * this->nBatchSize 7680 * 4320 * 3
```

Comment:
@Sampath1117, Dinesh was facing a segmentation fault with this line. After removing this->nBatchSize, the segmentation fault was resolved. Could you please review this?

Reply:
@HazarathKumarM That issues may be present only in SVC build machine . I reverted this change . I conformed with sampath It works fine in MI250 and MI300 machines .